adding_species_manuscript.tex

```diff
@@ -315,7 +311,7 @@ \section*{The utility of \stdpopsim for genome-wide simulations}
 and if necessary with input of additional members of the community.
 This quality control process quite often finds subtle bugs \citep[e.g., as in][]{Ragsdale2020}
 or highlights parts of the model that are ambiguously defined by the literature sources.
-This increases the reliability of the resulting simulations in any downstream analysis.
+This increases the reproducibility of the resulting simulations in any downstream analysis.
 
```

Comment:
Ah, okay. I was thinking about "reliable" as meaning that they'd, like, reliably run and not crash or something. Let's do "reliability and reproducibility". (since it is more reproducible if you know exactly what they're doing)